vm.go

```diff
@@ -538,8 +543,13 @@ func (session *ioSession) SendTerminalSize(columns, rows int) error {
 
 // SendSignal
 func (session *ioSession) SendSignal(signal syscall.Signal) error {
+	containerID := session.containerID
```

Comment:
There's still a problem, I think. if we have a shim for an exec (not a container workload) we can't forward the signal to the containerdID the exec belongs to.

Maybe, if containerID is "", we just return?